#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_fix_template_location.dpatch by  <cajus@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix location of configuration template.

@DPATCH@
diff -uNr gosa-core/include/functions.inc gosa-core/include/functions.new
--- gosa-core/include/functions.inc       2009-12-03 11:23:55.000000000 +0100
+++ gosa-core/include/functions.new       2009-12-03 11:26:16.000000000 +0100
@@ -40,7 +40,7 @@
 }

 /* Define common locatitions */
-define ("CONFIG_TEMPLATE_DIR", "../contrib");
+define ("CONFIG_TEMPLATE_DIR", "/usr/share/doc/gosa");
 define ("TEMP_DIR","/var/cache/gosa/tmp");

 /* Define get_list flags */

